x-pack/legacy/plugins/uptime/public/components/functional/location_map/__tests__/location_status_tags.test.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { renderWithIntl } from 'test_utils/enzyme_helpers';
+import { MonitorLocation } from '../../../../../common/runtime_types/monitor';
+import { LocationStatusTags } from '../';
+
+describe('StatusByLocation component', () => {
+  let monitorLocations: MonitorLocation[];
+
```

Comment:
I'm worried this could potentially break CI in the future - we're relying on `moment` to render these timestamps as relative values. Will they always render as `4d`, or will it change to `5d` tomorrow?

If you agree this is a concern, we could add some code like this:

```ts
const start = moment('2020-01-06T12:22:32.567Z');
beforeAll(() => {
  moment.prototype.fromNow = jest.fn((date: string) => start.from(date));
});
```

It's also an opportunity to test that our custom `locale` is rendering ok; we can specify a different `timestamp` to attempt to cover our most important ranges (i.e. `2s ago`, `4m ago`, `3h ago`, `9d ago`, etc.).

WDYT?

Reply:
"It's also an opportunity to test that our custom locale is rendering ok; we can specify a different timestamp to attempt to cover our most important ranges (i.e. 2s ago, 4m ago, 3h ago, 9d ago, etc.)."

That will require more effort, we will have to override function before each test. We can think about this, how can we automate the function for various useCases. Maybe write some sort of HOC testing component to handle this.

Reply:
Aha - I see what you mean. We'd have to modify the behavior and do a `beforeEach` to make sure we provide the appropriate timestamps in the `jest.fn` handler. That's ok for now then.